locales/fr.json

```diff
@@ -74,6 +74,7 @@
   },
   "collections": {
     "general": {
+      "all_of_collection": "Tous les {{ collection }}",
```

Comment:
Good call. I'll go with *All* for French and Spanish and leave the others intact. Sound good?